apps/web/client/src/app/test-sync-engine/_components/file-explorer.tsx

```
@@ -0,0 +1,315 @@
+import { ScrollArea } from '@onlook/ui/scroll-area';
+import { cn } from '@onlook/ui/utils';
+import { Input } from '@onlook/ui/input';
+
+export interface FileNode {
+    name: string;
+    path: string;
+    type: 'file' | 'directory';
+    children?: FileNode[];
+}
+
+export interface ContextMenuItem {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ContextMenuItem'. Consider to delete it or rename it.

'ContextMenuItem' is defined here:


(lint/suspicious/noRedeclare)